### PR TITLE
update - final touches completed

### PR DESCRIPTION
**Pull request by tinatsina**
## Summary
- finished React todolist
- used both `class-based `and `function-based `components

### CODE REVIEW

File: src/functionBased/components/TodoContainer.js [hunk 1]
```diff
@@ -65,4 +65,4 @@ const TodoContainer = () => {
   )
 }
 
-export default TodoContainer
\ No newline at end of file
+export default TodoContainer
```
Comment (PRATAP-KUMAR) [inline src/functionBased/components/TodoContainer.js:68]: line number 60, while clicking on the delete button, the item is not deleted.

deleteButton

**Review by PRATAP-KUMAR — CHANGES_REQUESTED** [review]
### Hi @tinatsina,

### Good job!

### Highlights
---
- The app is opening correctly on the browser.

### Required Changes :recycle:
---

_Check the comments under the review_

- Nice job on the implementation of the todo app.
- linters are not showing for this commit, can you check on this?
- presently the README file is from the CRA, It would be nice if you use the Microverse Template for this.

#### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding! :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


[OPTIONAL] you can check and implement the localStorage for the persistence of the todo items even after refreshing the page.

image

File: src/functionBased/components/TodoContainer.js (hunk 1 above)
Comment (tinatsina) [reply src/functionBased/components/TodoContainer.js:68]: ## Resolved 👍 
- had not yet changed the `todoList` to a functional component yet.
- changed it and refactored it
- now works fine
- thanks for noticing 👏

**Comment by PRATAP-KUMAR** [comment]
Hi, @tinatsina Good Job on Implementing the previous comments,

encourage

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! @tinatsina 

### Highlights
:heavy_check_mark: README is professional
:heavy_check_mark: linters are passing
:heavy_check_mark: the app is working

**Review by PRATAP-KUMAR — APPROVED** [review]

**Pull request merged**